api/modules/publishedProjects/model.js

```diff
@@ -0,0 +1,31 @@
+var BaseModel = require('../../classes/base_model');
+
+var Projects = require('../projects/model');
+
+var instanceProps = {
+  tableName: 'publishedProject',
```

Comment:
nit: as a general convention in sql (I don't know about postgres), table names are usually `_` delimited and not camel-cased